### ISSUE TITLE
AMD Support

### ISSUE THREAD

**Issue by gz** [gz/rust-cpuid#26]
https://support.amd.com/TechDocs/24594.pdf Appendix E

**Comment by daym** [comment]
It would also be nice (and easy) to support AMD family and model numbers.

See also https://www.amd.com/system/files/TechDocs/25481.pdf --important parts are quoted here:

So, if vendor is AuthenticAMD, family and model are actually these:

```
Family[7:0] = ({0000b,BaseFamily[3:0]} + ExtendedFamily[7:0].
If BaseFamily[3:0] is less than Fh then ExtendedFamily[7:0] is reserved and Family is equal to BaseFamily[3:0].

Model[7:0] = {ExtendedModel[3:0],BaseModel[3:0]}. For example,
if ExtendedModel[3:0] = Eh and BaseModel[3:0] = 8h, then Model[7:0] = E8h.
If BaseFamily[3:0] is less than 0Fh then ExtendedModel[3:0] is reserved and Model is equal to BaseMo
del[3:0].
```

IMO it's fine to just call them amd_family and amd_vendor in order to be explicit.

**Comment by gz** [comment]
This issue is largely fixed with #55, making individual issues for unsupported leafs that can be added with minor revisions to v10.
